DEVELOPMENT.md

```diff
@@ -8,6 +8,8 @@ Our goal is to build a repository where resources can be uploaded and shared, an
 
 We are reaching out to the open source community to help us with our mission.
 
+Our [Code of Conduct](https://github.com/QueerGlobal/qg-docs/main/CODE_OF_CONDUCT.md) covers this documentation as well as any other communication with QueerGlobal or any people involved with QueerGlobal
```

Comment:
I’m not sure why, but when I go to the link directly it doesn’t work. Is this on my end?

This link works though https://github.com/QueerGlobal/qg-docs/blob/8046d0ba951ddad32687b91c63311c713af3a1d0/CODE_OF_CONDUCT.md

Reply:
Hmm. I believe that I'll need to update this link.